src/test/scala/uk/gov/hmrc/test/ui/pages/AuthorityWizard.scala

```diff
@@ -26,8 +27,11 @@ object AuthorityWizard extends BasePage {
     this
   }
 
+  def navigateTo(url: String): Unit     = driver.navigate().to(url)
+  def findElementBy(by: By): WebElement = driver.findElement(by)
+
   def fillInputs(eoriNumber: String): this.type = {
-    val redirectionUrl = TestConfiguration.url("manage-transit-movements-frontend")
+    val redirectionUrl = "http://localhost:9949/auth-login-stub/session"
```

Comment:
Why is this restricted to localhost?

Reply:
Hard coded for the purpose of getting something passing last night. I will move to config and correct. 👍